utils/crypto.ts

```diff
@@ -164,3 +164,29 @@ export const electrumVaultMnemonicToSeed = (mnemonic: string, password = '') =>
     keylen: 64,
     digest: 'sha512',
   });
+
+export const getMasterPublicKeyPrefix = (header: string) => {
+  const headersTestnet = {
+    xpub: '043587cf',
+    ypub: '044a5262',
+    Ypub: '024289ef',
+    zpub: '045f1cf6',
+    Zpub: '02575483',
+  };
+  const headersMainnet = {
+    xpub: '0488b21e',
+    ypub: '049d7cb2',
+    Ypub: '0295b43f',
+    zpub: '04b24746',
+    Zpub: '02aa7ed3',
+  };
+
+  const getKeyByValue = (obj: HeaderObject) =>
+    Object.fromEntries(Object.entries(obj).map(([key, value]) => [key, value]));
+
+  if (config.networkName === 'bitcoinvault') {
+    return getKeyByValue(headersMainnet)[header];
```

Comment:
`headersMainnet[header]` there is no need to use a function, it seems that it doesn't have an influence on the object

Reply:
done